config.json

```diff
@@ -461,4 +472,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
This change is unnecessary to the subject of the PR.

*(Also, a CI check should probably have failed here. Considering how CI didn't catch a syntax error, perhaps `configlet fmt .` isn't run at all. I'll open an issue to investigate.)*